pulpcore/pulpcore/app/viewsets/repository.py

```diff
@@ -287,6 +245,27 @@ def create(self, request, repository_pk):
         return OperationPostponedResponse([result], request)
 
 
+class RemoteFilter(filterset.FilterSet):
+    """
+    Plugin remote filter would need:
+     - to inherit from this class
+     - to add any specific filters if needed
```

Comment:
- add type-specific filters 

If specific filters are not needed, they needn't subclass.